lib/fog/libvirt/compute.rb

```diff
@@ -57,12 +57,16 @@ def initialize(options={})
           require 'libvirt'
         end
 
-        private
-
         def client
-          return @client if defined?(@client)
+          @client ||= Class.new do
+            def libversion
+              1002009
```

Comment:
Yes, I have to admit I struggled a bit trying to understand how best to work with Shindo tests, have you any suggestions on how to mock/fake that the test there should be testing the code as though it's using a libvirt library version greater than 1.2.8 (1002009 corresponds to 1.2.9)?